src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-router.ts

```diff
@@ -13,4 +13,4 @@ JiraConnectEnterpriseAppRouter.post("/", JiraContextJwtTokenMiddleware, JiraConn
 JiraConnectEnterpriseAppRouter.route("/:uuid")
 	.get(csrfMiddleware, JiraJwtTokenMiddleware, JiraConnectEnterpriseAppCreateOrEdit)
 	.put(csrfMiddleware, JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPut)
-	.delete(csrfMiddleware, JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppDelete);
\ No newline at end of file
+	.delete(JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppDelete);
```

Comment:
This middleware only works on GET requests. We need to investigate this separately.